content/learn/getting-started.pt.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Getting Started
+mobile_menu_title: "Getting Started"
+toc: true
+---
+
+{{% div class="box thin"%}}
+**<center>Nota para usuários do Apple Silicon</center>**
+Zig tem suporte experimental para a assinatura de códigos. Você poderá usar Zig com seu M1 Mac,
+mas a única maneira de conseguir Zig para macOS em ARM64 é compilá-lo você mesmo.
+Certifique-se de verificar a seção [Compilando da Fonte](#building-from-source).
+{{% /div %}}
+
+
+## Marcar (Tags) as versões de lançamento e nightly?
+Zig ainda não atingiu a v1.0 e o atual ciclo de liberação está ligado a novas liberações de LLVM, que têm uma cadência de ~6 meses.
+Em termos práticos, **As versões Zig tendem a ser muito distantes e eventualmente se tornam obsoletas, dada a velocidade atual de desenvolvimento**.
+
+É bom avaliar o Zig usando uma versão com tags, mas se você decidir que gosta do Zig e quer mergulhar mais fundo, **nós o encorajamos a atualizar para versão nightly**, principalmente porque dessa forma será mais fácil para você obter ajuda: a maioria da comunidade e sites como
+[ziglearn.org](https://ziglearn.org) se baseiam no *master branch* pelas razões acima expostas.
+
+A boa notícia é que é muito fácil mudar de uma versão Zig para outra, ou mesmo ter várias versões presentes no sistema ao mesmo tempo: As versões Zig são arquivos autocontidos que podem ser colocados em qualquer lugar em seu sistema.
+
+
+## Instalando Zig
+### Baixar diretamente
+Esta é a maneira mais direta de obter o Zig: pegue um pacote Zig para sua plataforma a partir da página [Downloads](/download),
```

Comment:
```suggestion
Esta é a maneira mais direta de obter o Zig: pegue um pacote Zig para sua plataforma a partir da página [Baixar](../../download),
```

Please double check I got the relative URL right

Reply:
BTW, is `../../` better than `/pt/`?
Personally, /pt is easier to search if it changed in future

Reply:
The best would be to use a `ref` shortcode with a lang parameter, I think all other ways are hackish, but I don't want to annoy translators about this kind of refactoring, I'll do it once we are done with both your translations :)